ember_debug/deprecation-debug.js

```diff
@@ -206,7 +206,7 @@ export default EmberObject.extend(PortMixin, {
       // For ember-debug testing we usually don't want
       // to catch deprecations
       if (!this.get('namespace').IGNORE_DEPRECATIONS) {
-        this.get('deprecationsToSend').pushObject(deprecation);
+        this.get('deprecationsToSend').push(deprecation);
```

Comment:
```suggestion
        this.deprecationsToSend.pushObject(deprecation);
```

Is probably a better fix, the issue is `get` is entangling the array, and then we're immediately updating it. We still want to use the KVO compliant method if it's an Ember array, but we don't want to entangle.